### DISCUSSION TITLE
Null status/error code from GetChatMessageContentAsync exception

### DISCUSSION THREAD

**Discussion by Rohit-Juluru** [microsoft/semantic-kernel#9349] [Q&A] [answered]
Hello,

When ChatCompletion service is down or inaccessible, GetChatMessageContentAsync() exception returns a null status code. It gives out an AggregateException (for the use case I am testing locally with an incorrect URL) and all of the inner exception(s) do not have any error/status codes. The exception message has a status code and trying to get a status code from the message doesn't sound reliable.

Trying to failover to another region when there is an exception and handle use cases based on status codes.

Thanks.

**Comment by moonbox3** [comment]
Tagging @markwallace-microsoft for visibility.

**Comment by markwallace-microsoft** [comment, marked as answer]
@Rohit-Juluru I've created an issue for this and the development team will take a look: https://github.com/microsoft/semantic-kernel/issues/9381
